.github/workflows/test.yml

```diff
@@ -21,14 +21,18 @@ jobs:
             target: arm-unknown-linux-gnueabihf
             use-cross: true
 
-          - os: windows-latest
-            target: x86_64-pc-windows-msvc
+          - os: macos-latest
+            target: x86_64-apple-darwin
             use-cross: false
 
           - os: macos-latest
-            target: x86_64-apple-darwin
+            target: aarch64-apple-darwin
             use-cross: false
```

Comment:
I think also just comment this out for now to get CI passing and I'll sort things out with it later